docs/approach.rst

```diff
@@ -35,6 +33,8 @@ manner.
 
 Adaptive mask generation
 ````````````````````````
+:func:`tedana.utils.make_adaptive_mask`
```

Comment:
This matches the pattern employed in fMRIPrep's documentation, and is just slightly prettier, IMO.